superset/utils/webdriver.py

```diff
@@ -45,6 +45,13 @@
 WindowSize = tuple[int, int]
 logger = logging.getLogger(__name__)
 
+# Installation message for missing Playwright (needed for WebGL/DuckGL support)
+PLAYWRIGHT_INSTALL_MESSAGE = (
+    "To complete the migration from Cypress and enable WebGL/DuckGL screenshot "
```

Comment:
DuckGL :) a mutant animal born from the union DuckDB and DeckGL :)

Reply:
hehe deckgl

Reply:
<img width="1298" height="342" alt="Image" src="https://github.com/user-attachments/assets/603a919d-a59b-498c-a17b-69cf83a7ceee" /> i blame Fellow :D 😅